subprojects/instant-execution/src/main/kotlin/org/gradle/instantexecution/initialization/InstantExecutionStartParameter.kt

```diff
@@ -105,5 +101,5 @@ class InstantExecutionStartParameter(
 
     private
     fun systemProperty(propertyName: String) =
-        startParameter.systemPropertiesArgs[propertyName] ?: System.getProperty(propertyName)
+        startParameter.systemPropertiesArgs[propertyName]
```

Comment:
This was a hack to read system properties set from a TAPI client VM. This was used to be able to persist configuration in IDEs when only using system properties. This change removes that hack as it now possible again to configure the IDE to always pass command line arguments to Gradle (something that was absent in some IntelliJ versions over last year) and also because it is now possible to persistently enable the configuration cache using gradle properties in `gradle.properties`.